### PR TITLE
Use govuk-link, govuk-list and govuk-body styles from govuk-frontend

### PR DESCRIPTION
**Pull request by risicle**
https://trello.com/c/Oi60uTxu

Trying to tread the line between converting too much and not enough.

### CODE REVIEW

File: app/templates/brief.html [hunk 1]
```diff
@@ -104,8 +104,8 @@
 <div class="grid-row">
   <div class="column-two-thirds">
     <header class="page-heading-smaller">
-      <p class="context">{{ brief.organisation }}</p>
-      <h1>{{ brief.title }}</h1>
+      <p class="govuk-caption-l">{{ brief.organisation }}</p>
```
Comment (lfdebrux) [inline app/templates/brief.html]: This should be a span?

```suggestion
      <span class="govuk-caption-l">{{ brief.organisation }}</p>
```

Comment (risicle) [reply app/templates/brief.html]: Oh yes it should.

Thanks, fixed.

**Comment by lfdebrux** [comment]
These changes require a small fix to the functional tests to pass, I'll do that now.

**Comment by lfdebrux** [comment]
Functional tests are fixed, this is ready to merge

**Comment by risicle** [comment]
Thanks

**Pull request merged**